.github/workflows/master.yml

```diff
@@ -38,23 +38,62 @@ jobs:
           path: |
             packages/*/dist
             packages/viewer/loader
-  detect-publish:
+
+  detect-publish-release:
     runs-on: ubuntu-latest
     outputs:
-      publish: ${{ steps.detect_publish.outputs.publish }}
+      publish: ${{ steps.detect-publish.outputs.publish }}
     steps:
       - name: "Checkout changes"
         uses: actions/checkout@v2
       - name: "Detect publish"
-        id: "detect_publish"
+        id: "detect-publish"
         run: |
-          result=`./scripts/detect_publish.sh`
+          result=`./scripts/detect_release.sh`
           echo $result
           echo ::set-output name=publish::$result
-  publish:
+
+  publish-canary:
```

Comment:
Will this job run for each PR merged including release PRs? Just wondering if this job should depend on `detect-publish-release`, and have a `if: needs.detect-publish-release.outputs.publish != 1` conditional

Reply:
Yes, that's the intended behavior. When a release is run, I wanted to have the next canary also published that shadows the released version. This way if you're on a canary tag, and do an update, you'll effectively get the latest published version.